### PR TITLE
Update readme with latest info from docs page

### PR DESCRIPTION
**Pull request by dnth**
## Describe changes
I updated the README with new info from updated docs.

## Pre-requisites
Please ensure you have done the following:
- [x] I have read the **CONTRIBUTING.md** document.
- [ ] If my change requires a change to docs, I have updated the documentation accordingly.
- [ ] If I have added an integration, I have updated the [integrations](https://docs.zenml.io/advanced-guide/integrations) table and the [corresponding website section](https://zenml.io/integrations).
- [ ] I have added tests to cover my changes.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Other (add details above)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -294,24 +281,49 @@ def mnist_pipeline(
...
+
+
+# 📜 ZenFiles
+
+ZenFiles are production-grade ML use-cases powered by ZenML. They are fully
+fleshed out, end-to-end, projects that showcase ZenML's capabilities. They can
+also serve as a template from which to start similar projects.
+
+The ZenFiles project is fully maintained and can be viewed as a sister
+repository of ZenML. Check it out [here](https://github.com/zenml-io/zenfiles).
```
Comment (fa9r) [inline README.md:326]: Would remove that sentence, IMO it's not quite true, ZenFiles is the least-well-maintained code we have I think 😅 

```suggestion
Check out ZenFiles [here](https://github.com/zenml-io/zenfiles)!
```

File: README.md [hunk 2]
```diff
@@ -381,6 +411,24 @@ image](https://repobeats.axiom.co/api/embed/635c57b743efe649cadceba6a2e6a956663f
...
+
+
+We will host the gathering every Wednesday 8:30AM PT (5:30PM CET). 
+Register now through [this link](https://www.eventbrite.com/e/zenml-meet-the-community-tickets-354426688767), 
+or subscribe to the [public events calendar](https://calendar.google.com/calendar/u/0/r?cid=Y19iaDJ0Zm44ZzdodXBlbnBzaWplY3UwMmNjZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t) to get notified 
+before every community gathering.
+
 # 🆘 Where to get help
 
 First point of call should be [our Slack group](https://zenml.io/slack-invite/).
```
Comment (fa9r) [inline README.md:434]: I think you should also mention creating a GitHub issue as an alternative here

File: README.md [hunk 3]
```diff
@@ -329,27 +341,44 @@ zenml server up
...
+| 💬 **[Join Slack]** | Need help with your specific use case? Say hi on Slack! |
+| 🗺 **[Roadmap]** | See where ZenML is working to build new features. |
+| 🙋‍♀️ **[Contribute]** | How to contribute to the ZenML project and code base. |
 
-ZenFiles are production-grade ML use-cases powered by ZenML. They are fully
-fleshed out, end-to-end, projects that showcase ZenML's capabilities. They can
-also serve as a template from which to start similar projects.
+[ZenML 101]: https://docs.zenml.io/
+[Core Concepts]: https://docs.zenml.io/getting-started/core-concepts
+[API Guide]: https://docs.zenml.io/v/docs/developer-guide/steps-and-pipelines/functional-vs-class-based-api
```
Comment (fa9r) [inline README.md:367]: I think this one should not be here, it's not such a important concept IMO

**Review by fa9r — CHANGES_REQUESTED** [review]
Really great job, feels much more clear and much more approachable now, love it! 👏 👏 👏 

The only big request I have is to rework or remove the code example. IMO the README is not the right place for this. I would recommend to link to some actual example (quickstart / zenbytes 1 / ...) instead. If you wanted to highlight specific things with the example, e.g., the `@step()` and `@pipeline` syntax, then I would suggest to demonstrate that with short pseudo-code instead. 

My other comments are just minor suggestions, will also approve if those are not changed.

Really looking forward to get this PR in, let me know once it's ready for re-review 😊

**Review by strickvl — COMMENTED** [review]
Some small comments / nits on this. Otherwise I agree that I think it's time to reevaluate what we do with the code example in the README.

File: README.md [hunk 4]
```diff
@@ -381,15 +318,33 @@ image](https://repobeats.axiom.co/api/embed/635c57b743efe649cadceba6a2e6a956663f
...
-# 🆘 Where to get help
+# 👩‍👩‍👧‍👦 Meet the Team
+
+![Meet the Team](./docs/book/assets/meet_the_team.jpeg)
+
+Have a question that's too hard to express on our Slack? Is it just too much effort to say everything on a 
+long GitHub issue? Or are you just curious about what ZenML has been up to in the past week? Well, register now for the ZenML Office (Half) Hour to get your answers and more!
+It's free and open to everyone.
+
+Every week, part of the ZenML [core team](https://zenml.io/company#CompanyTeam) will pop in for 30 minutes to interact directly with the community. Sometimes we'll be presenting a feature. Other times just take questions and have fun. Join us if you are curious about ZenML, or just want to talk shop about MLOps.
```
Comment (fa9r) [inline README.md:329]:
```suggestion
Every week, part of the ZenML [core team](https://zenml.io/company#CompanyTeam) will pop in for 30 minutes to interact directly with the community. Sometimes we'll be presenting a feature. Other times we just take questions and have fun. Join us if you are curious about ZenML, or just want to talk shop about MLOps.
```

File: README.md [hunk 5]
```diff
@@ -381,15 +318,33 @@ image](https://repobeats.axiom.co/api/embed/635c57b743efe649cadceba6a2e6a956663f
...
+We will host the gathering every Wednesday 8:30AM PT (5:30PM CET). 
+Register now through [this link](https://www.eventbrite.com/e/zenml-meet-the-community-tickets-354426688767), 
+or subscribe to the [public events calendar](https://calendar.google.com/calendar/u/0/r?cid=Y19iaDJ0Zm44ZzdodXBlbnBzaWplY3UwMmNjZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t) to get notified 
+before every community gathering.
+
+# 🆘 Getting Help
+
+The first point of call should be [our Slack group](https://zenml.io/slack-invite/).
+Ask your questions about bugs or specific use cases, and someone from the [core team](https://zenml.io/company#CompanyTeam) will respond.
+Or if you prefer, [open an issue](https://github.com/zenml-io/zenml/issues/new/choose) on our GitHub repo.
```
Comment (fa9r) [inline README.md:342]:
```suggestion
Or, if you prefer, [open an issue](https://github.com/zenml-io/zenml/issues/new/choose) on our GitHub repo.
```

File: README.md [hunk 6]
```diff
@@ -1,355 +1,291 @@
...
 
-ZenML is built to support teams working together. The underlying infrastructure
-on which your ML workflows run can be shared, as can the data, assets and
-artifacts that you need to enable your work. ZenML Profiles offer an easy way to
-manage and switch between your stacks. The ZenML Server handles all the
-interaction and sharing and you can host it wherever you'd like.
+ZenML is built to support teams working together. 
+The underlying infrastructure on which your ML workflows run can be shared, as can the data, assets, and artifacts in your workflow. 
+
+You can quickly share your stack with anyone with ZenML by exporting them with:
```
Comment (fa9r) [inline README.md:231]: The concept of "stack" has not been introduced yet, maybe you can add a short intro.

```suggestion
In ZenML, a Stack represents a set of configurations for your MLOps tools and infrastructure. You can quickly share your ZenML stack with anyone by exporting the stack:
```

File: README.md [hunk 7]
```diff
@@ -366,7 +302,8 @@ influence the roadmap as follows:
   board](https://zenml.io/discussion). You can also request for new features here.
 - Start a thread in our [Slack channel](https://zenml.io/slack-invite).
```
Comment (fa9r) [inline README.md:303]: Maybe also mention "create github issue" here? @strickvl or is that even a recommended way of requesting features? I remember we discussed at some point whether all feature requests should go over the roadmap instead, what was the decision on this again?

**Review by fa9r — APPROVED** [review]
Nice, looks great! I left a few more minor suggestions, but I think you could merge as is. Well done 👍

**Pull request merged**